doc/man3/SSL_CONF_cmd.pod

```diff
@@ -441,6 +452,15 @@ middleboxes that do not understand TLSv1.3 will not drop the connection. This
 option is set by default. A future version of OpenSSL may not set this by
 default. Equivalent to B<SSL_OP_ENABLE_MIDDLEBOX_COMPAT>.
 
+B<AntiReplay>: If set then OpenSSL will automatically detect if a session ticket
+has been used more than once, TLSv1.3 has been neotiated and early data is
+enabled on the server. A full handshake is forced if a session ticket is used a
+second or subsequent time. This option is set by default and is only used by
+servers. Anti-replay measures are required to comply with the TLSv1.3
```

Comment:
Now I see, the previous comment resulted from cut/paste.  Why do we have both `AntiReplay` and `-no_anti_replay`?

Reply:
SSL_CONF supports two modes of configuring stuff - command line args and configuration file commands. Almost all options appear in two forms: one for each.